From: Michael Froman <mjfroman@mac.com>
Date: Fri, 17 May 2024 21:01:17 -0500
Subject: (tmp-cherry-pick) Revert "Add flag to exclude policy exempt field
 trials when listing expired ones" (868ab5e9a8)

This reverts commit 47cfed2a7db30b775b8c386a4374b04d6bcbf5a6.

Reason for revert: breaking CI

Original change's description:
> Add flag to exclude policy exempt field trials when listing expired ones
>
> Bug: None
> Change-Id: I07bc9f3ad1172bcdaf205937fb518ec295c022bf
> No-Try: True
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/346420
> Commit-Queue: Emil Lundmark <lndmrk@webrtc.org>
> Reviewed-by: Jeremy Leconte <jleconte@google.com>
> Cr-Commit-Position: refs/heads/main@{#42019}

Bug: None
Change-Id: Idba5a521c2a9b2ad2327452295093204db7b2cf3
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/346440
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Auto-Submit: Jeremy Leconte <jleconte@google.com>
Commit-Queue: Jeremy Leconte <jleconte@google.com>
Cr-Commit-Position: refs/heads/main@{#42020}
---
 experiments/field_trials.py | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/experiments/field_trials.py b/experiments/field_trials.py
index 4bf2e62fef..8274c03a8d 100755
--- a/experiments/field_trials.py
+++ b/experiments/field_trials.py
@@ -1027,9 +1027,7 @@ def cmd_header(args: argparse.Namespace) -> None:
 def cmd_expired(args: argparse.Namespace) -> None:
     today = todays_date()
     diff = datetime.timedelta(days=args.in_days)
-    expired = expired_field_trials(
-        today + diff,
-        REGISTERED_FIELD_TRIALS if args.exempt else ACTIVE_FIELD_TRIALS)
+    expired = expired_field_trials(today + diff)
 
     if len(expired) <= 0:
         return
@@ -1074,11 +1072,6 @@ def main() -> None:
         Lists all expired field trials. Exits with a non-zero exit status if
         any field trials has expired, ignoring the --in-days argument.
         ''')
-    parser_expired.add_argument(
-        '--exempt',
-        default=True,
-        action=argparse.BooleanOptionalAction,
-        help='whether to include policy exempt field trials')
     parser_expired.add_argument(
         '--in-days',
         default=0,
